Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs

```diff
@@ -507,6 +529,162 @@ private byte[] GetKey(ShortHash hash)
             return hash.ToByteArray();
         }
 
+        #region Memoization
+        // TODO(jubayard): garbage collection / removal in general
+
+        /// <inheritdoc />
+        public override GetContentHashListResult GetContentHashList(OperationContext context, StrongFingerprint strongFingerprint)
+        {
+            var key = GetCaChaaSKey(strongFingerprint);

```

Comment:
cachaas or cazaas?

Reply:
CaChaaS for sure! See Sergey's comment on the Discussion

---
In reply to: [302164434](https://github.com/microsoft/BuildXL/pull/571#discussion_r302164434) [](ancestors = 302164434)